php/php-tarantool/php-tarantool-dev.spec

```diff
@@ -23,9 +23,7 @@
 
 %global github_owner     tarantool
 %global github_name      tarantool-php
-%global github_commit    27697cf6c1bf0d87a46ac6412767d46aa04ce5f3
-%global github_short     %(c=%{github_commit}; echo ${c:0:7})
-%global github_date      20160906
+%global github_version   0.2.0
```

Comment:
Thanks for this PR.
BTW, I always use the commit ref in all case (mostly because some upstream sometime badly move their tags...)